translate/convert/l20n2po

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
```

Comment:
This file is no longer necessary. `translate/convert/l20n2po.py` does the trick.